ykrt/src/compile/jitc_yk/codegen/x86_64.rs

```diff
@@ -638,6 +642,33 @@ impl<'a> X64CodeGen<'a> {
         Ok(())
     }
 
+    /// Codegen a (non-varargs) call.
+    pub(super) fn codegen_call_instr(
+        &mut self,
+        inst_idx: InstrIdx,
+        inst: &jit_ir::CallInstruction,
+    ) -> Result<(), CompilationError> {
+        let func_decl_idx = inst.target();
+        let func_type = func_decl_idx.func_type(self.jit_mod);
+        let args = (0..(func_type.num_args()))
```

Comment:
Can't we also use `inst.num_args` here? It should be the same as `func_type.num_args()`.

Reply:
You have a keen eye :)

We could, but I prefer it this way, as it emphasises that these are static arguments of the function type, not the (potentially dynamic, for a varargs call) arguments of a call-site.

Reply:
The type of the instruction differs, so you couldn't do it without adding traits.